privacyidea/lib/tokens/totptoken.py

```diff
@@ -675,31 +676,39 @@ def get_default_settings(cls, params, logged_in_user=None,
         ret = {}
         if not logged_in_user:
             return ret
+        (scope, username, userrealm, adminuser, adminrealm) = determine_logged_in_userparams(logged_in_user,
```

Comment:
Just an idea: if we return a dictionary here, we could pass it to the other functions without explicitly stating the parameter names. This could reduce the code a bit.

Reply:
I understand. But since this is only a part of the paramters taken by the policy  functions I think this could result in less readable code, since there would be some implicit functionality.

Reply:
Will not do it in this case.